nextflow.config

```diff
@@ -13,7 +13,7 @@ profiles {
   docker { 
     docker.enabled = true
     process {
-      container = 'haniffalab/webatlas-pipeline:0.5.2'
+      container = 'haniffalab/webatlas-pipeline:0.5.3'
```

Comment:
Please, also bump version in the following files:
- [envs/build-docker-imgs.sh](https://github.com/cellgeni/webatlas-pipeline/blob/517bc321756eeb33955fabb96da7effde83157e8/envs/build-docker-imgs.sh#L2)
- [main.nf](https://github.com/cellgeni/webatlas-pipeline/blob/517bc321756eeb33955fabb96da7effde83157e8/main.nf#L8)
- [multimodal.nf](https://github.com/cellgeni/webatlas-pipeline/blob/517bc321756eeb33955fabb96da7effde83157e8/multimodal.nf#L14)
- [sphinx/conf.py](https://github.com/cellgeni/webatlas-pipeline/blob/517bc321756eeb33955fabb96da7effde83157e8/sphinx/conf.py#L26)
- [sphinx/examples/visium.rst](https://github.com/cellgeni/webatlas-pipeline/blob/517bc321756eeb33955fabb96da7effde83157e8/sphinx/examples/visium.rst?plain=1#L153) (multiple places in file)
- [sphinx/examples/xenium.rst](https://github.com/cellgeni/webatlas-pipeline/blob/517bc321756eeb33955fabb96da7effde83157e8/sphinx/examples/xenium.rst?plain=1#L144) (multiple places in file)
- [sphinx/installation.rst](https://github.com/cellgeni/webatlas-pipeline/blob/517bc321756eeb33955fabb96da7effde83157e8/sphinx/installation.rst#L20) (multiple places in file)

Reply:
done now!